ports/stm/boards/STM32F411_nvm_flash.ld

```diff
@@ -0,0 +1,29 @@
+/*
+    GNU linker script for STM32F411 with nvm and an external flash chip, and reserves
+    more space for the CircuitPython firmware and less for the filesystem
+    (since the filesystem will be on the external flash chip).
+*/
+
+/* Specify the memory areas */
+MEMORY
+{
+    FLASH (rx)          : ORIGIN = 0x08000000, LENGTH = 512K /* entire flash */
+    FLASH_ISR (rx)      : ORIGIN = 0x08000000, LENGTH = 16K /* sector 0 */
+    FLASH_FS (rx)       : ORIGIN = 0x08004000, LENGTH = 32K /* sectors 1,2 are 16K */
```

Comment:
You can just remove this, since you default to an external flash.

Reply:
Awesome! I wasn't sure that would be possible...but it also seemed like a waste to reserve space that wouldn't be used.

Reply:
It still seems like a shame to me that NVM has to take up a full sector, but it just hasn't been high priority enough to spend the time to fix, especially when external flash makes it irrelevant anyway.